source/draw/RasterBgr.ooc

```diff
@@ -117,7 +117,7 @@ RasterBgr: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 3
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntSize2D new(x, y))
+		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents, true), IntSize2D new(x, y))
```

Comment:
I can't remember which function was it exactly, but there is one call in Windows API that you must do twice - first you pass a null buffer so it tells you how big buffer you need, then on second call you pass the non-null buffer pointer allocated with the size returned from first call. Simply beautiful isn't it ? :speak_no_evil: